addons/html_editor/static/src/core/content_editable_plugin.js

```diff
@@ -1,5 +1,5 @@
-import { isArtificialVoidElement } from "@html_editor/core/selection_plugin";
 import { Plugin } from "@html_editor/plugin";
+import { isMediaElement } from "@html_editor/utils/dom_info";
```

Comment:
This commit only includes the addition of the import I guess something more should be in this commit (the removal of isArtificialVoidElement) I guess